src/components/ButtonPanel.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './ButtonPanel.scss';
+import Button from './Button.jsx';
+
+function ButtonPanel() {
+  return (
```

Comment:
We should use `class function here` so that we can implement `render function` as mentioned in the instructions.
![image](https://user-images.githubusercontent.com/31897434/72196590-9c571180-3419-11ea-8340-299d221f6648.png)

class function example  👉  `class App extends React.Component {`